osmosis-apidb/src/main/java/org/openstreetmap/osmosis/apidb/v0_6/impl/ChangeWriter.java

```diff
@@ -369,7 +373,7 @@ public void write(Node node, ChangeAction action) {
 
         } catch (SQLException e) {
             throw new OsmosisRuntimeException(
-            		"Unable to check if current node with id=" + node.getId() + " exists.", e);
```

Comment:
Similar to earlier, centralise the whole exception creation, not just a bit of the string.